src/translations/dist/en.json

```diff
@@ -264,11 +363,7 @@
     "crampons_spring": {
       "glacier_gear_types": "crampons_spring"
     },
-    "Create a new article": "Create a new article",
```

Comment:
Why have these been removed? They still exist in the code.

Reply:
I see `add an article`, but not `Create a new article`